### DISCUSSION TITLE
17.2.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#943] [General]
## Releases Notes for 17.2.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/draw.io-17.2.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/draw.io-17.2.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/draw.io-universal-17.2.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/drawio-amd64-17.2.1.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/drawio-x86_64-17.2.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v17.2.1/drawio-x86_64-17.2.1.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Adds defensive code to try to deal with case of machine suddenly powering off and corrupting the saved file.
- Updates to [draw.io core 17.2.1](https://github.com/jgraph/drawio/blob/v17.2.1/ChangeLog). All changes from 16.5.2 to 17.2.1, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v17.2.1'>17.2.1</a>.</em>

**Comment by eupraxis1981** [comment]
as noted elsewhere, this version has a serious problem for Windows 10 -- extremely slow to load and operate. Not usable.
